### PR TITLE
Merge QRCodeInterface into AdminInterface 

### PR DESCRIPTION
**Pull request by Geod24**
CC @TrustHenry : Can you test this ?

### CODE REVIEW

File: source/agora/node/admin/AdminInterface.d [hunk 1]
```diff
@@ -68,47 +74,225 @@ public class AdminInterface
...
-        return listenHTTP(settings, router);
+    public override string encryptionKeyQR (
+        string app, ulong height, out string contentType, out string vary)
+    {
+        // TODO: PreImage to be released at that height
+        // Or it can be pre-released, so it can be `height + PreimageRevealPeriod`
+        PreImageInfo pre_image;
+
+        // blake2b hashing
+        const Hash key = hashMulti(pre_image, app);
```
Comment (hewison-chris) [inline source/agora/node/admin/AdminInterface.d:151]: Can it be pre_image.hash ?

Comment (Geod24) [reply source/agora/node/admin/AdminInterface.d:151]: I just moved the code. It always uses the default-initialized pre-image so it's not valid anyway...

**Comment by Geod24** [comment]
For reference, this was triggered because I am working on adding an endpoint to dynamically reconfigure the loggers via HTTP queries.

**Review by hewison-chris — APPROVED** [review]
LGTM

**Review by linked0 — APPROVED** [review]
LGTM. We need to reformat the docs of `NodeControlAPI` functions though.

File: source/agora/node/admin/AdminInterface.d (hunk 1 above)
Comment (TrustHenry) [reply source/agora/node/admin/AdminInterface.d:151]: This should be fixed to a block height-based pre-image later.

**Review by TrustHenry — APPROVED** [review]
LGTM

**Pull request merged**